database_admin/schema/create_schema.sql

```diff
@@ -426,6 +426,23 @@ BEGIN
 END;
 $fun$ LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION create_table_partitions(tbl regclass, parts INTEGER, rest text)
```

Comment:
Why the regclass type is used here?

Reply:
Only allows registered names like tables or indexes in the function.